.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "src/main/webapp/client"]
+	path = src/main/webapp/client
+	url = https://github.com/terrestris/momo3-frontend
+[submodule "src/main/webapp/lib/BasiGX"]
+	path = src/main/webapp/lib/BasiGX
+	url = https://github.com/terrestris/BasiGX
+[submodule "src/main/webapp/lib/geoext3"]
+	path = src/main/webapp/lib/geoext3
+	url = https://github.com/geoext/geoext3
```

Comment:
I allready changed this in the frontend. The frontend now shares BasiGX and GeoExt with our other ext apps.

https://github.com/terrestris/momo3-frontend/commit/5b5bfa7ad8b786d8371ecb4169914e29df01ec10